### PR TITLE
AT_04.02.009 |Breadcrumbs| Dashboard page link check Manage Jenkins d…

### PR DESCRIPTION
**Pull request by AlexSQE**
…ropdown subfolder
https://trello.com/c/348jNUhP/1173-at0402009-breadcrumbsdashboard-page-link-check-manage-jenkins-dropdown-subfolder

### CODE REVIEW

File: cypress/e2e/breadcrumbsMenu.cy.js [hunk 1]
```diff
@@ -63,5 +63,13 @@ describe('BreadcrumbsMenu', () => {
          expect($el.text()).to.be.equal(homePage.dashboardDropdownItems[idx])
       })
    })
+
+   it.only('AT_04.02.009 |Breadcrumbs| Dashboard page link check Manage Jenkins dropdown subfolder', () => {
```
Comment (DariaMas7) [inline cypress/e2e/breadcrumbsMenu.cy.js:67]: delete only,please

File: cypress/e2e/breadcrumbsMenu.cy.js [hunk 2]
```diff
@@ -63,5 +63,13 @@ describe('BreadcrumbsMenu', () => {
...
+      cy.get('#breadcrumbs .jenkins-menu-dropdown-chevron').realHover().realClick();
+      cy.get("#breadcrumb-menu [href='\/manage'] span").contains("Manage Jenkins").trigger('mouseover');
```
Comment (DariaMas7) [inline cypress/e2e/breadcrumbsMenu.cy.js:69]: "Manage Jenkins" move to fixtures, please

**Review by DariaMas7 — COMMENTED** [review]
Overall great job

File: cypress/e2e/breadcrumbsMenu.cy.js (hunk 1 above)
Comment (AlexSQE) [reply cypress/e2e/breadcrumbsMenu.cy.js:67]: Thanks!!!

File: cypress/e2e/breadcrumbsMenu.cy.js (hunk 2 above)
Comment (AlexSQE) [reply cypress/e2e/breadcrumbsMenu.cy.js]: Misfire)) Done

**Review by DimaCurkanu — APPROVED** [review]

**Review by S-Lanchik — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]

**Pull request merged**